src/writer.rs

```diff
@@ -1079,6 +1079,11 @@ impl<W: io::Write> Writer<W> {
         }
     }
 
+    /// Return a reference to the underlying writer.
+    pub fn get_ref(&self) -> &W {
+        self.wtr.as_ref().unwrap()
+    }
+
```

Comment:
Sorry, one more thing. Can you move this above `into_inner` to match the order of methods on `csv::Reader`? Thanks.